### DISCUSSION TITLE
Show output of the commands that are running

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [spatie/laravel-short-schedule#63] [Ideas] [answered]
When running the command `php artisan short-schedule:run`, it doesn't output what tasks are being run by the scheduler. 
The laravel command `php artisan schedule:run` does this. Any chance you can add this feature to this package?

**Comment by freekmurze** [comment, marked as answer]
That is currently not supported, but feel free to create a PR that adds this functionality.

**Reply by abishekrsrikaanth** [reply]
ok, let me take a dig at this.
